lib/ReactViews/ExplorerWindow/Tabs/MyDataTab/AddData.tsx

```diff
@@ -198,19 +217,21 @@ class AddData extends React.Component {
   renderPanels() {
     const { t } = this.props;
     const dropdownTheme = {
+      // @ts-expect-error FIXME: no Styles.dropdown defined.
```

Comment:
Need help with these, not sure why there isn't a `Style.dropdown` and `Style.dropdownListIsOpen` defined.